### PR TITLE
added collapsed-objectives-program-years component

### PR DESCRIPTION
**Pull request by djvoa12**
**Summary:**
Program years should now use `collapsed-objectives-program-years` component for the collapsed summary as they relate to competencies and not parents.

**UI:**
Screen Shot 2019-04-02 at 3 26 36 PM

Accompanies https://github.com/ilios/common/pull/608
Fixes https://github.com/ilios/frontend/issues/4463

### CODE REVIEW

File: app/templates/components/collapsed-objectives-program-years.hbs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+{{#if (and
+  (is-fulfilled this.objectives)
+  (is-fulfilled this.objectivesWithMesh)
+  (is-fulfilled this.objectivesWithCompentency))}}
+  <div class="content">
+    <table>
+      <thead>
+        <tr>
+          <th class="text-left">{{t "general.summary"}}</th>
+          <th class="text-center">{{t "general.linkedCompetencies"}}</th>
```
Comment (stopfstedt) [inline app/templates/components/collapsed-objectives-program-years.hbs]: please add this translation as well.

File: app/templates/components/collapsed-objectives-program-years.hbs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            {{else if (gte (get (await this.objectivesWithMesh) "length") 1)}}
+              {{fa-icon "circle" class="maybe"}}
+            {{else}}
+              {{fa-icon "ban" class="no"}}
+            {{/if}}
+          </td>
+        </tr>
+        <tr>
+          <td>
+            &nbsp;
```
Comment (stopfstedt) [inline app/templates/components/collapsed-objectives-program-years.hbs]: i don't get those non-breaking spaces. are you indenting here? if so - no need, get rid of them.

File: app/templates/components/collapsed-objectives-program-years.hbs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            &nbsp;
+            {{t "general.linkedCompetencyCount"
+              count=(get (await this.objectivesWithCompentency) "length")}}
+          </td>
+        </tr>
+        <tr>
+          <td>
+            &nbsp;
```
Comment (stopfstedt) [inline app/templates/components/collapsed-objectives-program-years.hbs]: see comment above, pls remove those.

File: app/templates/components/collapsed-objectives-program-years.hbs (hunk 2 above)
Comment (djvoa12) [reply app/templates/components/collapsed-objectives-program-years.hbs]: I didn't add this part. This was there before I started working on it.

**Comment by stopfstedt** [comment]
needs a new common release, after that this should be  good to go. @djvoa12 - i'll ping you when the time comes to rebase against master, so that tests will be passing.

**Pull request merged**